### PR TITLE
Add the first four mnemonic accounts 

### PR DESCRIPTION
**Pull request by epheph**
~75K eth, plenty to work with, not so much you destroy metamask (or other) interface

### CODE REVIEW

**Comment by epheph** [comment]
Maybe i should do first 10 for consistency?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# parity-instantseal
+Builds instantseal Docker container, normally available at keydonix/parity-instantseal.
+
+Comes funded with these accounts:
+- 0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb (private key: 0xfae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5an), loaded with 10^20 ETH
```
Comment (MicahZoltu) [inline README.md:5]: Do you mean 2^128 ETH (which is about 10^38.5)?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- These accounts from the mnemonic "zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong", all loaded with 100 ETH
```
Comment (MicahZoltu) [inline README.md]: It appears these are all loaded with 10^20 ETH, not 100 ETH

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    * (HD Path: m/44'/60'/0/0) - 0xfc2077CA7F403cBECA41B1B0F62D91B5EA631B5E
```
Comment (MicahZoltu) [inline README.md]: Are these the actual derivation paths you used?  If so, they are wrong (not to spec).  :)
```
m/44'/60'/0'/0/x
```

File: README.md (hunk 1 above)
Comment (epheph) [reply README.md:5]: yea, figure it was more useful for README to report it in ETH terms

File: README.md (hunk 2 above)
Comment (epheph) [reply README.md]: They are loaded with 100 ETH:
```
		"0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb": { "balance": "0xffffffffffffffffffffffffffffffff" },
		"0xfc2077ca7f403cbeca41b1b0f62d91b5ea631b5e": { "balance": "0x56bc75e2d63100000" },

```

File: README.md (hunk 1 above)
Comment (MicahZoltu) [reply README.md:5]: Ah, misread as attoeth.  👍

File: README.md (hunk 2 above)
Comment (MicahZoltu) [reply README.md]: Misread.  So used to attoeth.

**Review by MicahZoltu — APPROVED** [review]

**Pull request merged**